src/components/weather/weather-display.js

```diff
@@ -111,13 +114,13 @@ class ForecastedWeather extends Component {
 
         // precipitation
         if (hour.hasOwnProperty("precipType")) {
-          if (hour.hasOwnProperty("precipType")) {
-            if (forecastInfo.precipitation.startAt !== null) {
-              forecastInfo.precipitation.startAt = hour.time;
-              forecastInfo.precipitation.willPrecip = true;
-              forecastInfo.precipitation.type = hour.precipType;
-            }
-            forecastInfo.precipitation.amount += hour.precipAccumulation;
+          if (forecastInfo.precipitation.startAt !== null) {
+            forecastInfo.precipitation.startAt = hour.time;
+            forecastInfo.precipitation.type = hour.precipType;
+          }
+          forecastInfo.precipitation.amount += hour.precipAccumulation;
```

Comment:
Not sure why this line and the following one are outside of the previous if block. If the startAt time is not defined I don't think you want to define an amount of precipitation.